cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/StateVariablesExport.java

```diff
@@ -208,7 +207,7 @@ private static void writePowerFlows(Network network, String cimNamespace, XMLStr
 
         Map<String, Double> equivalentInjectionTerminalP = new HashMap<>();
         Map<String, Double> equivalentInjectionTerminalQ = new HashMap<>();
-        network.getDanglingLineStream().forEach(dl -> {
+        network.getDanglingLineStream().filter(dl -> !dl.isMerged()).forEach(dl -> {
```

Comment:
`network.getDanglingLineStream().filter(dl -> !dl.isMerged())` is done many times. We could add a
`network.getAloneDanglingLineStream()` ?

Reply:
Added a `getDanglingLineStream(DanglingLineFilter danglingLineFilter)` to `Network`, `Bus` and `VoltageLevel` interfaces